modules/combined/test/tests/gap_heat_transfer_mortar/gap_heat_transfer_mortar_displaced.i

```diff
@@ -25,33 +25,18 @@
   material_coverage_check = false
 []
 
-[AuxVariables]
+[Variables]
```

Comment:
How about a test with reference residual? Maybe you need #17249 first...

Reply:
Yes I would

Reply:
Second order meshes seem to show more convergence challenges for the cases I have tried.